docs/api/app.md

```diff
@@ -852,6 +852,19 @@ correctly.
 
 **Note:** This will not affect `process.argv`.
 
+### `app.getSystemAccentColor()` _Windows_
+
+Returns the users current system wide color preference in the form of an ARGB
+hexadecimal string.
```

Comment:
I would return it as RGBA hex to match browsers: http://stackoverflow.com/a/27802062/64949